src/app/workflow/executions/executions-tab.component.html

```diff
@@ -1,117 +1,188 @@
-          </td>
-        </ng-container>
+          <ng-container matColumnDef="average">
+            <th mat-header-cell *matHeaderCellDef>Average</th>
+            <td mat-cell *matCellDef="let executionMetric">
+              <span *ngIf="executionMetric.average" matTooltip="{{ executionMetric.average + executionMetric.unit }}"
+                >{{ executionMetric.average | number: '1.0-2' }}{{ executionMetric.unit }}</span
+              >
+            </td>
+          </ng-container>
 
-        <ng-container matColumnDef="max">
-          <th mat-header-cell *matHeaderCellDef>Maximum</th>
-          <td mat-cell *matCellDef="let element">
-            <span *ngIf="element.max">{{ element.max + element.unit }}</span>
-          </td>
-        </ng-container>
+          <ng-container matColumnDef="maximum">
+            <th mat-header-cell *matHeaderCellDef>Maximum</th>
+            <td mat-cell *matCellDef="let executionMetric">
+              <span *ngIf="executionMetric.maximum" matTooltip="{{ executionMetric.maximum + executionMetric.unit }}"
+                >{{ executionMetric.maximum | number: '1.0-2' }}{{ executionMetric.unit }}</span
+              >
+            </td>
+          </ng-container>
 
-        <tr mat-header-row *matHeaderRowDef="metricsColumns"></tr>
-        <tr mat-row *matRowDef="let row; columns: metricsColumns"></tr>
-      </table>
-    </div>
-  </div>
-  <div class="w-100 mt-5">
-    <table mat-table class="validations-table" aria-label="validations-table" [dataSource]="validationsTable" data-cy="validations-table">
-      <ng-container matColumnDef="validatorTool">
-        <th mat-header-cell *matHeaderCellDef>Validator Tool</th>
-        <td data-cy="metrics-validatorTool" mat-cell *matCellDef="let element">{{ element.validatorTool }}</td>
-      </ng-container>
+          <tr mat-header-row *matHeaderRowDef="executionMetricsColumns"></tr>
+          <tr mat-row *matRowDef="let row; columns: executionMetricsColumns"></tr>
+        </table>
+      </div>
+    </mat-card-content>
+  </mat-card>
+  <div class="mt-5">
+    <mat-card>
+      <mat-card-header>
+        <mat-card-title>Validator Tool Metrics</mat-card-title>
+      </mat-card-header>
+      <mat-divider></mat-divider>
+      <mat-card-content class="pt-3">
+        <div *ngIf="!validatorToolMetricsExist">No validator tool metrics exist</div>
+        <div *ngIf="validatorToolMetricsExist">
+          <mat-form-field data-cy="metrics-validator-tool-dropdown">
+            <mat-label>Select a Validator Tool</mat-label>
+            <mat-select [value]="currentValidatorTool" (selectionChange)="onSelectedValidatorToolChange($event.value)">
+              <mat-option data-cy="metrics-validator-tool-options" [value]="validatorTool" *ngFor="let validatorTool of validatorTools">
+                {{ validatorTool }}
+              </mat-option>
+            </mat-select>
+          </mat-form-field>
+          <div fxFlexLayout="row" fxLayoutGap="2rem">
```

Comment:
WebStorm giving a warning -- shouldn't this be `fxLayout` instead of `fxFlexLayout`?